src/sql/parts/modelComponents/toolbarContainer.component.ts

```diff
@@ -29,7 +29,7 @@ export class ToolbarItem {
 	template: `
 		<div #container *ngIf="items" [class]="toolbarClass" >
 			<ng-container *ngFor="let item of items">
-			<div class="modelview-toolbar-item" [title]="getItemTitle(item)" [style.paddingTop]="paddingTop" tabindex="0">
```

Comment:
Why get rid of tabIndex? I think it was likely added for accessibility - can you check with other folks on this?

Reply:
Each component is a control and it is tabable. By doing this, you have 2 layers of tab

---
In reply to: [216546738](https://github.com/Microsoft/sqlopsstudio/pull/2488#discussion_r216546738) [](ancestors = 216546738)